packages/react-router-dom/modules/NavLink.js

```diff
@@ -43,11 +45,13 @@ NavLink.propTypes = {
   className: PropTypes.string,
   activeStyle: PropTypes.object,
   style: PropTypes.object,
-  isActive: PropTypes.func
+  isActive: PropTypes.func,
+  ariaCurrent: PropTypes.oneOf(['page', 'step', 'location', 'true'])
 }
 
 NavLink.defaultProps = {
-  activeClassName: 'active'
+  activeClassName: 'active',
+  ariaCurrent: 'true'
```

Comment:
Should this be `page` by default? That seems to be the most common use case within a routing library.